Runtime/Scripts/SelectionTools/PolygonInput.cs

```diff
@@ -201,6 +215,12 @@ private void FinishAction_performed(InputAction.CallbackContext obj)
 
         public void ReselectPolygon(List<Vector3> points)
         {
+            if (mode == DrawMode.Create)
+            {
+                Debug.LogWarning("PolygonInput is in create mode, cannot reselect polygon in edit mode.", gameObject);
```

Comment:
Ik heb deze warnings er bewust in gelaten voor mogelijke debugging bij het gebruik van de package